sklearn/cross_decomposition/_pls.py

```diff
@@ -334,7 +334,7 @@ def transform(self, X, Y=None, copy=True):
 
         return x_scores
 
-    def inverse_transform(self, X):
+    def inverse_transform(self, X=None, Y=None):
```

Comment:
In principle, we do not need both inputs here, but making `X` optional would be a first for scikit-learn. I think we would still need keep `X` and have `Y` optional.

This would be similar to how `transform` works.

Reply:
Hello, thanks for having a look at this. I do agree with your comment. In my daily workflow, there's a step where I only need to inverse transform for Y, hence my implementation with optional X. However, since the operation is not computationally intensive, your review would solve my issue and better fit within scikit-learn.

Reply:
I agree with @thomasjpfan that it would be more natural to have always `X`.